src/org/traccar/model/Position.java

```diff
@@ -65,15 +66,26 @@
     public static final String PREFIX_ADC = "adc";
     public static final String PREFIX_IO = "io";
     public static final String PREFIX_COUNT = "count";
+    public static final String PREFIX_BUTTON = "button";
 
+    public static final String ALARM_GENERAL = "general";
     public static final String ALARM_SOS = "sos";
     public static final String ALARM_VIBRATION = "vibration";
     public static final String ALARM_MOVEMENT = "movement";
+    public static final String ALARM_LOW_SPEED = "lowspeed";
     public static final String ALARM_OVERSPEED = "overspeed";
     public static final String ALARM_FALL_DOWN = "fallDown";
     public static final String ALARM_LOW_BATTERY = "lowBattery";
     public static final String ALARM_MOTION = "motion";
     public static final String ALARM_FAULT = "fault";
+    public static final String ALARM_POWER_OFF = "powerOff";
+    public static final String ALARM_GEOFENCE = "geofence";
+    public static final String ALARM_GEOFENCE_ENTER = "geofenceEnter";
+    public static final String ALARM_GEOFENCE_EXIT = "geofenceExit";
+    public static final String ALARM_GPS_ANTENNA_CUT = "gpsAntennaCut";
+    public static final String ALARM_OIL_CUT = "oilCut";
```

Comment:
I don't think we need oil cut alarm. I'm guessing that it means operator sent the command to stop the engine.